todocore/src/main/java/com/knighttodo/todocore/exception/FindRoutineInstanceByCreationDateException.java

```diff
@@ -0,0 +1,12 @@
+package com.knighttodo.todocore.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.NOT_FOUND)
+public class FindRoutineInstanceByCreationDateException extends RuntimeException{
```

Comment:
RoutineInstanceByCreationDateNotFoundException? you shouldn't name classes starting with verbs, it's for methods.